deployment/secure_research_desktop/packages/packages-python.yaml

```diff
@@ -69,15 +61,9 @@ packages:
     statsmodels:
         "all": [">0.10.0"]  # increase solver flexibility
     sympy:
-    tables:
```

Comment:
Instead of removing `tables` could you mark it as `uninstallable` for all Python versions and explain why?

I'm not sure what dependency problems this packages would have.